app/views/content/international-teacher-training-to-english-standards-introducing-iqts.md

```diff
@@ -0,0 +1,177 @@
+---
+title: International qualified teacher status (iQTS)
+description: |-
+ International qualified teacher status (iQTS) is a new teaching qualification backed by England’s Department for Education (DfE). iQTS is equivalent to qualified teacher status (QTS). You can train where you live and work. 
+date: "2021-08-06"
+image: "media/images/content/hero-images/0003.jpg"
+backlink: "../"
+keywords: 
+    - international
+    - QTS
+    - iQTS
+    - Qualified teacher status
+---
+
+## Introducing iQTS
+
+International qualified teacher status (iQTS) is a new teaching qualification backed by England’s Department for Education (DfE). iQTS is equivalent to qualified teacher status (QTS). You can train where you live and work. 
```

Comment:
Might add a bit more info to the third sentence? Something along the lines of 'You can train towards your iQTS from where you live and work.' Might just make it read a bit better? And maybe make the second sentence just 'It is equivalent to qualified teacher status' to avoid too much repetition of iQTS.